From 5a34aa80b2d5d31fc46cea7f4f4c6ed59dcc140c Mon Sep 17 00:00:00 2001
From: chengyu_lin <chengyu_lin@aspeedtech.com>
Date: Wed, 17 Sep 2025 11:13:08 +0800
Subject: [PATCH 29/32] aspeed/clock: Modify clock selection lock

To support runtime uart driver clock selection modification,
do not lock the uart1~13 clock selection in scu policy driver.

Signed-off-by: chengyu_lin <chengyu_lin@aspeedtech.com>
Change-Id: I7fc8fe6f7bdbccf8822ea77942b84123d9caa65f
(cherry picked from commit be046b35782c3dd830ac5450e82d8f4c1702fa45)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 board/aspeed/ibex_ast2700/sys_policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/aspeed/ibex_ast2700/sys_policy.c b/board/aspeed/ibex_ast2700/sys_policy.c
index 214210f5f7..af7986f2bf 100644
--- a/board/aspeed/ibex_ast2700/sys_policy.c
+++ b/board/aspeed/ibex_ast2700/sys_policy.c
@@ -25,7 +25,7 @@
 #define SYS_POLICY_CLK0_SEL1_LOCK GENMASK(15, 0)
 #define SYS_POLICY_CLK0_SEL2_LOCK GENMASK(12, 0)
 #define SYS_POLICY_CLK0_SEL3_LOCK GENMASK(1, 0)
-#define SYS_POLICY_CLK1_SEL1_LOCK (GENMASK(14, 0) | BIT(18) | BIT(21) | \
+#define SYS_POLICY_CLK1_SEL1_LOCK (GENMASK(14, 13) | BIT(18) | BIT(21) | \
 				   BIT(25) | BIT(29))
 #define SYS_POLICY_CLK1_SEL2_LOCK (BIT(0) | BIT(3) | BIT(8) | BIT(12) | \
 				   GENMASK(20, 15) | BIT(23))
-- 
2.34.1

